docs/api-ref.md

```diff
@@ -2683,10 +2682,59 @@ The schedule methods are based upon the endpoints for schedules in the REST API
 
 Source files: server/endpoint/schedules_endpoint.py
 
+#### schedule.add_to_schedule
+
+```py
+schedules.add_to_schedule(schedule_id)
+```
+
+Adds a `DataSourceItem` or a `WorkbookItem` to a schedule.
```

Comment:
This should be DatasourceItem (lower case s).

Reply:
Done